src/submit_jobs.py

```diff
@@ -60,49 +60,49 @@ def parse_args(defaults) -> argparse.Namespace:
         "--json-file",
         type=str,
         nargs="?",
-        help="""
-        Path to a JSON file containing job configuration(s)". The path is
-        relative to the project `config/` directory
-        """,
```

Comment:
Kind of interesting: I'm pretty sure this is auto-formatting from ruff designed to escape the double quote. I definitely wouldn't have used triple quotes here.